### PR TITLE
fmt(cstyle): fixes in src/istgt_iscsi_xcopy.h

### PR DESCRIPTION
**Pull request by bharathpgp**
Fixed improper first line of block comment, line > 80 characters, missing blank before close comment.

### CODE REVIEW

File: src/istgt_iscsi_xcopy.h [hunk 1]
```diff
@@ -59,12 +62,13 @@ typedef struct istgt_xcopy_tgt_t {
 } ISTGT_XCOPY_TGT;
 
 int istgt_lu_disk_receive_copy_results(CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd);
-int istgt_lu_disk_xcopy(ISTGT_LU_DISK *spec, CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd);
-int istgt_lu_disk_process_xcopy(ISTGT_LU_DISK *spec, CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd, uint8_t *data_buf, int dlen);
-int istgt_lu_disk_lbxcopy(ISTGT_XCOPY_TGT *src_tgt, ISTGT_XCOPY_TGT *dst_tgt, CONN_Ptr conn,
-		ISTGT_LU_CMD_Ptr lu_cmd, int dc, int cat,  uint64_t num_blks_byts, uint8_t sd_opcode);
+int istgt_lu_disk_xcopy(ISTGT_LU_DISK *spec, \
```
Comment (vishnuitta) [inline src/istgt_iscsi_xcopy.h]: we don't need 'slashes' in function definitions.. similarly, at other places of function definitions

File: src/istgt_iscsi_param.c [hunk 2]
```diff
@@ -81,39 +81,40 @@ istgt_iscsi_param_del(ISCSI_PARAM **params, const char *key)
...
 int
-istgt_iscsi_param_add(ISCSI_PARAM **params, const char *key, const char *val, const char *list, int type)
+istgt_iscsi_param_add(ISCSI_PARAM **params, const char *key, const char *val,
+	const char *list, int type)
 {
 	ISCSI_PARAM *param, *last_param;
 
 	ISTGT_TRACELOG(ISTGT_TRACE_DEBUG, "add %s=%s, list=[%s], type=%d\n",
-				   key, val, list, type);
+	key, val, list, type);
```
Comment (vishnuitta) [inline src/istgt_iscsi_param.c]: we need to have 4 spaces here

**Comment by bharathpgp** [comment]
_**Output of C style run:**_
**bharath@debian:~/github/istgt$** ./cstyle.pl src/istgt_iscsi_param.c
**bharath@debian:~/github/istgt$** ./cstyle.pl src/istgt_iscsi_xcopy.h
**bharath@debian:~/github/istgt$** ./cstyle.pl src/ring_mempool.h

File: src/istgt_iscsi_param.c [hunk 3]
```diff
@@ -81,39 +81,40 @@ istgt_iscsi_param_del(ISCSI_PARAM **params, const char *key)
 
 	ISTGT_TRACELOG(ISTGT_TRACE_DEBUG, "del %s\n", key);
 	if (params == NULL || key == NULL)
-		return 0;
+		return (0);
 	for (param = *params; param != NULL; param = param->next) {
-		if (param->key != NULL && param->key[0] == key[0]
-			&& strcasecmp(param->key, key) == 0) {
+		if (param->key != NULL && (param->key[0] == key[0]) &&
+		strcasecmp(param->key, key) == 0) {
```
Comment (vishnuitta) [inline src/istgt_iscsi_param.c]: 4 spaces would be required here

File: src/istgt_iscsi_param.c [hunk 4]
```diff
@@ -64,14 +64,14 @@ istgt_iscsi_param_find(ISCSI_PARAM *params, const char *key)
...
 
 	if (params == NULL || key == NULL)
-		return NULL;
+		return (NULL);
 	for (param = params; param != NULL; param = param->next) {
-		if (param->key != NULL && param->key[0] == key[0]
-			&& strcasecmp(param->key, key) == 0) {
-			return param;
+		if (param->key != NULL && param->key[0] == key[0] &&
+			strcasecmp(param->key, key) == 0) {
```
Comment (vishnuitta) [inline src/istgt_iscsi_param.c]: here, we need to have strcasecmp 4 spaces away under 'if'

File: src/istgt_iscsi_xcopy.h [hunk 5]
```diff
@@ -59,12 +62,13 @@ typedef struct istgt_xcopy_tgt_t {
 } ISTGT_XCOPY_TGT;
 
 int istgt_lu_disk_receive_copy_results(CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd);
-int istgt_lu_disk_xcopy(ISTGT_LU_DISK *spec, CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd);
-int istgt_lu_disk_process_xcopy(ISTGT_LU_DISK *spec, CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd, uint8_t *data_buf, int dlen);
-int istgt_lu_disk_lbxcopy(ISTGT_XCOPY_TGT *src_tgt, ISTGT_XCOPY_TGT *dst_tgt, CONN_Ptr conn,
-		ISTGT_LU_CMD_Ptr lu_cmd, int dc, int cat,  uint64_t num_blks_byts, uint8_t sd_opcode);
+int istgt_lu_disk_xcopy(ISTGT_LU_DISK *spec,
+	CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd);
```
Comment (vishnuitta) [inline src/istgt_iscsi_xcopy.h]: lets have 4 spaces instead of tab in second line (for below function definitions as well)

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**